chart/values.yaml

```diff
@@ -214,6 +214,17 @@ parquetMetadata:
   # Directory on the shared storage (parquet metadata files used for random access in /rows)
   storageDirectory: "/parquet-metadata"
 
+duckDBIndex:
+  # Directory on the shared storage (used temporarily to prepare the duckdb indexes before sending to the Hub)
+  storageDirectory: "/duckdb-index"
```

Comment:
Using a non-shared temporary directory would also have worked no ?

I'm fine with using a shared storage though

Reply:
I think it's due to the size (the pods have nearly no space, while the shared storage has as much space as we need (EFS))

Reply:
(even if, by default, the local duckdb will be small)